repo/packages/R/ruxit/0/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "packagingVersion": "2.0",
+    "name": "ruxit",
+    "version": "0.1",
+    "tags": [ "ruxit", "apm", "monitoring" ],
+    "description": "Cloud-native monitoring powered by artificial intelligence",
+    "maintainer": "alois.mayr@dynatrace.com",
+    "framework": false,
+    "licenses": [
+        {
+            "name": "MIT License",
+            "url": "https://github.com/Dynatrace/Dynatrace-Docker/blob/master/LICENSE"
+        }
+    ],
+    "postInstallNotes": "Thank you for installing Dynatrace Ruxit on DCOS.\nLog in to your Dynatrace Ruxit account at https://login.ruxit.com and monitor your DCOS cluster.",
+    "postUninstallNotes": "Dynatrace Ruxit has been uninstalled from DCOS and will no longer run.\n.",
+    "preInstallNotes": "Pass your environment ID (ruxit.environment) and token (ruxit.token) as package options.\nAlso pass the number of Mesos slaves as an option (ruxit.instances).\nMarathon will make sure that the Ruxit Agent runs only once per slave.",
```

Comment:
That's all fine. I'd like to have the default resources (cpu/memory) listed here as well.